source/meetings/2022/april/index.html.md

```diff
@@ -0,0 +1,82 @@
+---
+updated_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+created_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+category: meeting
+title: April 2022 Meeting
+published_at: 2022-04-04 21:30:00 +0000
+created_at: 2022-04-04 20:51:00 +0000
+status: Published
+---
+
+The April 2022 meeting of LRUG will be on *Monday the 11th of April*,
+from _6:20pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+[Full registration details are given below](#apr22registration), but do
+note that **we will only be sending out the zoom meeting url to people who
+have registered, because we have limited places, so please do it.**
+
+## Agenda
+
+### Using React in a Ruby Project to Dynamically Generate PDF Documents
+
+[Panos Matsinopoulos](https://getlavanda.com) says:
+
+
+> Fjust needs to be a couple of sentences really, not a
+> full abstract. We use markdown so feel free to format it or embed links for
+> the writeup.
+> 
```

Comment:
I believe you may have forgotten to delete these lines. May you confirm?

Reply:
yeah overly eager copypasta